agent/agent_test.go

```diff
@@ -853,7 +853,7 @@ func TestAgent_CheckAliasRPC(t *testing.T) {
 	assert.NoError(t, err)
 
 	retry.Run(t, func(r *retry.R) {
-		t.Helper()
+		r.Helper()
```

Comment:
are you sure this is correct? looks like `r.Helper()` is a no-op, so may as well just delete it?

Reply:
The implementation of r.Helper is:

```
func (r *R) Helper() {
	// *testing.T will just record which functions are helpers by their addresses and
	// it doesn't much matter where where we record that they are helpers
	r.wrapped.Helper()
}
```

Its passed through to the main `*testing.T` because its just recording func addresses that should be considered helpers